### DISCUSSION TITLE
Cycle vie des données

### DISCUSSION THREAD

**Discussion by PhilippeLRETQC** [MTESSDev/FRW#114] [Annonces]
Bonjour,

Nous regardons le cycle de vie des données de la solution et l'épuration dans les systèmes.

L'état "Épuré par le traitement lot, formulaire trop vieux" est bien le cas qu'un formulaire a dépassé la date d'expiration sans être transmis?

Quand un formulaire est rendu à un des états suivants : "Supprimé par l'utilisateur", "Transmis (soumis) par l'utilisateur" ou "Épuré par le traitement lot, formulaire trop vieux". Combien de temps sont conservées les informations du formulaire et les pièces jointes au MESS? Est-ce qu'il y a un moyen de demander la destruction des données si le délai était trop long?

Merci beaucoup,
Philippe

**Comment by loima02** [comment]
Bonjour,

> L'état "Épuré par le traitement lot, formulaire trop vieux" est bien le cas qu'un formulaire a dépassé la date d'expiration sans être transmis?

Notre traitement d'épuration roule quotidiennement pour épurer les formulaires. Le nombre de jours varie selon certains critères et peut être ajusté pour répondre à nos besoins sans préavis. Voici les paramètres actuels :
* Nombre de jours avant d'épurer un formulaire anonyme non transmis : 90
* Nombre de jours avant d'épurer un formulaire authentifié non transmis : 180
* Nombre de jours avant d'épurer un formulaire supprimé par l'utilisateur : 1
* Nombre de jours avant d'épurer un formulaire transmis : 90

> Est-ce qu'il y a un moyen de demander la destruction des données si le délai était trop long?

En deux mots... Non. Si ça devient un besoin, il faudra soumettre une suggestion d'amélioration.

**Reply by PhilippeLRETQC** [reply]
Merci beaucoup, les délais me semble très acceptable.
